kornia/contrib/models/structures.py

```diff
@@ -104,3 +104,10 @@ def keypoints(self) -> Tensor | None:
     def keypoints_labels(self) -> Tensor | None:
         """The keypoints labels from the `points`"""
         return self.points[1] if isinstance(self.points, tuple) else None
+
+
+@dataclass
+class DetectionResults:
+    labels: Tensor
+    scores: Tensor
+    bboxes: Tensor
```

Comment:
Should we use Boxes here to give more semantics? @johnnv1 @shijianjian

Reply:
In future I would like though that we standardised to something like 
```python 
class BoundingBox:
     x: float
     y: float
     width: float
     height: float

 @dataclass
 class Detection:
     bounding_box: BoundingBox
     confidence: float
     class_id: int
```

so that can be directly used too with protobuf without the need to serialize entire tensors. Planning at some point to have a kornia-proto that basically will be our structures definition reference

Reply:
also, what should be the box format? I think xyxy should be good? `(x1, y1, x2, y2)` (top left and bottom right)

Reply:
the `Boxes` class now (kornia.geoemtry.boxes.Boxes) supports conversion between the formats/modes, so this can help too. `xyxy` or `xyhw` it's fine for me, i think COCO format is on `xyhw` which can be a good foothold as long most the detections models use datasets on COCO format

Reply:
Shall I wrap the bounding boxes with `kornia.geoemtry.boxes.Boxes`?

Reply:
I think so, for the visual prompter, I have done this https://github.com/kornia/kornia/blob/master/kornia/contrib/visual_prompter.py#L151 because for general users raw tensor can be easy to understand than our data structure

Reply:
FYI, paddle detection uses (labels / class id, score, bbox) format: https://github.com/PaddlePaddle/PaddleDetection/blob/develop/ppdet/modeling/post_process.py#L563-L568

Just let me know which format/wrapper you prefer, I will change accordingly.